Gems/SceneProcessing/Code/Source/Config/Components/SceneProcessingConfigSystemComponent.cpp

```diff
@@ -139,6 +149,54 @@ namespace AZ
             return success;
         }
 
+        void SceneProcessingConfigSystemComponent::LoadScriptSettings()
+        {
+            auto* registry = AZ::SettingsRegistry::Get();
+            AZ_Assert(registry, "AZ::SettingsRegistryInterface should already be active!");
+
+            auto* fileIO = AZ::IO::FileIOBase::GetInstance();
+            AZ_Assert(fileIO, "AZ::IO::FileIOBase should already be active!");
+
+            using VisitArgs = AZ::SettingsRegistryInterface::VisitArgs;
+            using VisitResponse = AZ::SettingsRegistryInterface::VisitResponse;
+            using VisitAction = AZ::SettingsRegistryInterface::VisitAction;
+            auto vistor = [this, registry, fileIO](const VisitArgs& args, VisitAction action) -> VisitResponse
+            {
+                if (action == VisitAction::Begin)
+                {
+                    return VisitResponse::Continue;
+                }
+                else if (action == VisitAction::End)
+                {
+                    return VisitResponse::Done;
+                }
+                else if (args.m_type != AZ::SettingsRegistryInterface::Type::String)
+                {
+                    return VisitResponse::Continue;
+                }
+
+                AZStd::string value;
+                registry->Get(value, args.m_jsonKeyPath);
+                auto fullPath = fileIO->ResolvePath(value.c_str());
+                if (!fullPath || !fileIO->Exists(fullPath->c_str()))
+                {
+                    AZ_Warning("SceneProcessing", false, "Could not resolve script path %s", value.c_str());
```

Comment:
Can this be a little more detailed, if someone sees "Could not resolve script path objects/processing.py" they won't really know what that's coming from.  Maybe: "<setting name> references script %s which could not be resolved or does not exist on disk"

Reply:
updated